test/tsickle_test.ts

```diff
@@ -62,6 +62,9 @@ function compareAgainstGolden(output: string|null, path: string) {
       }
     }
   } else {
+    // Make sure we have proper line endings when testing on Windows.
+    if (golden != null) golden = golden.replace(/\r\n/g, '\n');
+    if (output != null) output = output.replace(/\r\n/g, '\n');
```

Comment:
This seems like the wrong place for this.  What about the above check on line 51 for output !== golden?  I think this logic should move up to before line 51 perhaps.